java/Nikiforov-Oleksandr/src/com/zoolatech/lecture1/tasks/_2/Task2.java

```diff
@@ -0,0 +1,26 @@
+package com.zoolatech.lecture1.tasks._2;
+import java.util.Scanner;
+
+/**
+ Write a program that accepts a distance in meters and time in seconds and outputs a speed
+ in meters/second and kilometers/hour. The result should be a floating point number.
+ */
+
+public class Task2 {
+    public static void main(String[] args) {
+        // task solution code here
+        Scanner sc = new Scanner(System.in);
+        float distance = sc.nextFloat();
+        float time = sc.nextFloat();
+        calculateSpeed(distance,time);
+    }
+
+
+    public static void  calculateSpeed(float distance, float time){
+        // calculate speed in m/sec  and in km/h
```

Comment:
similar to a task 1: the comment can be removed if you define values as variables